### ISSUE TITLE
Problematic release (4.14.1) : version.txt: No such file or directory

### ISSUE THREAD

**Issue by laulaz** [Simplistix/testfixtures#60]
```
Processing testfixtures-4.14.1.tar.gz
Unpacking testfixtures-4.14.1 to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1
Unpacking testfixtures-4.14.1/testfixtures to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures
Unpacking testfixtures-4.14.1/testfixtures/tests to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests
Unpacking testfixtures-4.14.1/testfixtures/tests/test_django to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_django
Unpacking testfixtures-4.14.1/testfixtures/tests/test_django/__init__.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_django/__init__.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_django/models.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_django/models.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_django/settings.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_django/settings.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_django/tests.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_django/tests.py
Unpacking testfixtures-4.14.1/testfixtures/tests/__init__.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/__init__.py
Unpacking testfixtures-4.14.1/testfixtures/tests/compat.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/compat.py
Unpacking testfixtures-4.14.1/testfixtures/tests/sample1.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/sample1.py
Unpacking testfixtures-4.14.1/testfixtures/tests/sample2.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/sample2.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_compare.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_compare.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_comparison.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_comparison.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_components.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_components.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_date.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_date.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_datetime.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_datetime.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_diff.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_diff.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_docs.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_docs.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_generator.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_generator.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_log_capture.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_log_capture.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_logcapture.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_logcapture.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_manuel.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_manuel.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_manuel_examples.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_manuel_examples.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_outputcapture.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_outputcapture.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_popen.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_popen.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_popen_docs.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_popen_docs.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_rangecomparison.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_rangecomparison.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_replace.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_replace.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_replacer.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_replacer.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_roundcomparison.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_roundcomparison.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_should_raise.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_should_raise.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_shouldwarn.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_shouldwarn.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_stringcomparison.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_stringcomparison.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_tempdir.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_tempdir.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_tempdirectory.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_tempdirectory.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_time.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_time.py
Unpacking testfixtures-4.14.1/testfixtures/tests/test_wrap.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tests/test_wrap.py
Unpacking testfixtures-4.14.1/testfixtures/__init__.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/__init__.py
Unpacking testfixtures-4.14.1/testfixtures/comparison.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/comparison.py
Unpacking testfixtures-4.14.1/testfixtures/compat.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/compat.py
Unpacking testfixtures-4.14.1/testfixtures/components.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/components.py
Unpacking testfixtures-4.14.1/testfixtures/django.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/django.py
Unpacking testfixtures-4.14.1/testfixtures/logcapture.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/logcapture.py
Unpacking testfixtures-4.14.1/testfixtures/manuel.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/manuel.py
Unpacking testfixtures-4.14.1/testfixtures/outputcapture.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/outputcapture.py
Unpacking testfixtures-4.14.1/testfixtures/popen.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/popen.py
Unpacking testfixtures-4.14.1/testfixtures/replace.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/replace.py
Unpacking testfixtures-4.14.1/testfixtures/resolve.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/resolve.py
Unpacking testfixtures-4.14.1/testfixtures/rmtree.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/rmtree.py
Unpacking testfixtures-4.14.1/testfixtures/shouldraise.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/shouldraise.py
Unpacking testfixtures-4.14.1/testfixtures/shouldwarn.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/shouldwarn.py
Unpacking testfixtures-4.14.1/testfixtures/tdatetime.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tdatetime.py
Unpacking testfixtures-4.14.1/testfixtures/tempdirectory.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/tempdirectory.py
Unpacking testfixtures-4.14.1/testfixtures/utils.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/utils.py
Unpacking testfixtures-4.14.1/testfixtures.egg-info to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures.egg-info
Unpacking testfixtures-4.14.1/testfixtures.egg-info/PKG-INFO to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures.egg-info/PKG-INFO
Unpacking testfixtures-4.14.1/testfixtures.egg-info/SOURCES.txt to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures.egg-info/SOURCES.txt
Unpacking testfixtures-4.14.1/testfixtures.egg-info/dependency_links.txt to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures.egg-info/dependency_links.txt
Unpacking testfixtures-4.14.1/testfixtures.egg-info/not-zip-safe to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures.egg-info/not-zip-safe
Unpacking testfixtures-4.14.1/testfixtures.egg-info/requires.txt to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures.egg-info/requires.txt
Unpacking testfixtures-4.14.1/testfixtures.egg-info/top_level.txt to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures.egg-info/top_level.txt
Unpacking testfixtures-4.14.1/README.rst to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/README.rst
Unpacking testfixtures-4.14.1/setup.py to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/setup.py
Unpacking testfixtures-4.14.1/PKG-INFO to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/PKG-INFO
Unpacking testfixtures-4.14.1/setup.cfg to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/setup.cfg
Reading configuration from /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/setup.cfg
Adding new section [easy_install] to /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/setup.cfg
Writing /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/setup.cfg
Running testfixtures-4.14.1/setup.py bdist_egg --dist-dir /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/egg-dist-tmp-bzoLBW
error: /var/folders/vD/vDv5nbN4E281TopcFoq6CU+++TI/-Tmp-/easy_install-rMp6QT/testfixtures-4.14.1/testfixtures/version.txt: No such file or directory

```

**Comment by cjw296** [comment]
How very strange.

I'll look into this, but how come you're not using the wheel?

**Comment by cjw296** [comment]
Fixed in 4.13.3.
